cmd/server/main.go

```diff
@@ -28,15 +28,20 @@ func main() {
 			Usage: "ocid pod sandbox dir",
 		},
 		cli.StringFlag{
-			Name:  "runtime",
-			Value: "/usr/bin/runc",
-			Usage: "OCI runtime path",
-		},
-		cli.StringFlag{
 			Name:  "containerdir",
 			Value: "/var/lib/ocid/containers",
 			Usage: "ocid container dir",
 		},
+		cli.StringFlag{
+			Name:  "socket",
+			Value: unixDomainSocket,
+			Usage: "path to ocid socket",
+		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Value: "/usr/bin/runc",
```

Comment:
Can we make this `/usr/sbin/runc` which is the "default" install location for runC? Or do other distributions put this in `/usr/bin`?